queries.sql

```diff
@@ -8,3 +8,29 @@ select name,escape_attempts from animals where weight_kg > 10.5;
 select * from animals where neutered = true;
 select * from animals where name <> "Gabumon";
 select * from animals where weight_kg >= 10.4 and weight_kg <= 17.3;
+select * from animals;
+UPDATE ANIMALS SET species = 'unspecified';
+select * from animals;
+ROLLBACK;
+select * from animals;
+delete from animals;
+select * from animals;
+rollback;
+select * from animals;
```

Comment:
- [ ] Please make the transactions here as mentioned in the project requirements as seen in the screenshot below. Currently, you are missing most of them
![image](https://user-images.githubusercontent.com/45460192/228693655-b82dfd0d-14bf-4a8c-bd23-48171e424bfb.png)

Reply:
added all